CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
       - Fixed an issue that would detect turning circles as sliproads
       - Fixed a bug where post-processing instructions (e.g. left + left -> uturn) could result in false pronunciations
       - Fixes a bug where a bearing range of zero would cause exhaustive graph traversals
+    - Infrastructure:
+      - Adds a feature to limit results in nearest service with a default of 100 in `osrm-routed`
```

Comment:
Yes, this is fine :+1: 

What does not work is e.g. linking VROOM against `libosrm` v5.3. And then upgrading `libosrm` to v5.4. If you want to upgrade `libosrm` you have to compile/link it in again.

For the record, I started a C wrapper for FFI bindings and ABI stability some time ago, just in case someone later reads this and needs those guarantees: https://github.com/daniel-j-h/libosrmc